App/Services/EventHandlers/NotificationEventHandler.ts

```diff
@@ -0,0 +1,38 @@
+import { Store } from 'redux'
+import PushNotification from 'react-native-push-notification'
+import {PushNotificationIOS} from 'react-native'
+import { RootState } from '../../Redux/Types'
+import TriggersActions from '../../Redux/TriggersRedux'
+import * as TextileTypes from '../../Models/TextileTypes'
+
+export default class NotificationEventHandler {
+  store: Store<RootState>
+
+  constructor(store: Store<RootState>) {
+    this.store = store
+    this.setup()
+  }
+
+  onNotification (notification: TextileTypes.NotificationEngagement) {
+    if (notification.userInteraction === true) {
+      this.store.dispatch(TriggersActions.newEngagement(notification))
+    }
+    if (notification.finish) {
+      notification.finish(PushNotificationIOS.FetchResult.NoData)
```

Comment:
Not sure what this does, but just flagging as Android TODO since it's using an iOS only API